docker/docker-compose.yml

```diff
@@ -196,6 +196,8 @@ services:
       - ELASTIC_MAX_RESULTS_PER_PAGE
       - ELASTIC_URL=http://elasticsearch
       - ELASTIC_PORT=9200
+      - SERVICE_ROLE_KEY
```

Comment:
So this can probably be the anon key only.

Reply:
Changed to `SERVICE_ROLE_KEY` because of this comment https://github.com/Seneca-CDOT/telescope/pull/3465#discussion_r848646880